index.html

```diff
@@ -559,6 +559,22 @@ <h3>Use Cases and Requirements</h3>
 or <a>verifiers</a>, because the conformance of ecosystem roles are highly
 application, use case, and market vertical specific.
         </p>
+
+        <p>
+While digital proofs, a subset of which are digital signatures, are required
+to ensure the protection of a <a>verifiable credential</a>, this specification
+does not standarize on any single digital signature format. At the time of
+publication, the Working Group included members that had implemented using
+at least three proof mechanisms. These implementations utilized
+JSON Web Tokens, Linked Data Signatures, and Camenisch-Lysyanskaya
+Zero-Knowledge Proofs. It is expected that each of these mechanisms will mature
+independently and additional mechanisms will also become standardized. One
+of the goals of this specification is to provide a data model that can be
+protected by a variety of current and future digital proof approaches.
+Conformance with respect to digital proofs in this specification is focused
+on the clear identification of each digital proof mechanism and not the details
+of any particular proof mechanism.
```

Comment:
"Conformance to this specification does not depend on the details of a particular proof mechanism; it requires clearly identifying the mechanism a verifiable credential uses."?

Reply:
Fixed in 276007376a0082128439f109c32779c50f740656.